static/app/components/feedback/list/feedbackListItem.tsx

```diff
@@ -54,6 +54,7 @@ const FeedbackListItem = forwardRef<HTMLDivElement, Props>(
     const hasReplayId = feedbackHasReplay(feedbackItem.id);
 
     const isCrashReport = feedbackItem.metadata.source === 'crash_report_embed_form';
+    const isUserReportWithError = feedbackItem.metadata.source === 'user_report_envelope';
```

Comment:
Some people might think it's good to mess around up here and combine things because we're looking at `feedbackItem.metadata.source` in both cases, and then looking at the two `is*` variables down below...

I like it the way it is. It's clear that these are two things, and the condition below is easy to read.